addons/medical_gui/initKeybinds.inc.sqf

```diff
@@ -23,7 +23,7 @@
     };
 
     // Check conditions: canInteract and canOpenMenu
-    if !([ACE_player, _target, ["isNotInside", "isNotSwimming"]] call EFUNC(common,canInteractWith)) exitWith {false};
+    if !([ACE_player, _target, ["isNotInside", "isNotSwimming", "isNotEscorting", "isNotDragging", "isNotCarrying", "isNotOnLadder"]] call EFUNC(common,canInteractWith)) exitWith {false};
```

Comment:
are changes these reversed?
Don't we want more the expectation on peek, not the medical menu

Reply:
My bad, I must have fat-fingered the wrong line 🤦🏻‍♂️